tools/create_f2fs_image.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+#run this script in syzkaller/tools dir
+declare -a Op1=("-a 0 " "-a 1 ")
+declare -a Op2=("-s 1 " "-s 2 ")
+declare -a Op3=("-m " "")
+declare -a Op4=("-O encrypt " "-O compression " "")
+declare -i dex=0
+
+for op1 in "${Op1[@]}"; do
+  for op2 in "${Op2[@]}"; do
+    for op3 in "${Op3[@]}"; do
+      for op4 in "${Op4[@]}"; do
+        echo mkfs.f2fs ${op1}${op2}${op3}${op4} disk.raw
+        fallocate -l 64M disk.raw
+        mkfs.f2fs "${op1}""${op2}""${op3}""${op4}" disk.raw 
+        go run "./syz-imagegen/imagegen.go" -image=./disk.raw -fs=f2fs > ../sys/linux/test/syz_image_mount_f2fs_$dex
```

Comment:
Using the "./" here requires running from a certain CWD.  Can you either document that requirement, or make it automatically work from any CWD (e.g., by using `dirname $0`).